tests/ImageSharp.Tests/Image/ImageTests.Decode_Cancellation.cs

```diff
@@ -16,65 +16,105 @@ public class Decode_Cancellation : ImageLoadTestBase
 
         public Decode_Cancellation() => this.TopLevelConfiguration.StreamProcessingBufferSize = 128;
 
-        public static readonly TheoryData<string> TestFiles = new()
+        private static TheoryData<bool, string, double> GetTestData()
```

Comment:
It's never occured to me to generate theory data in this manner. Big chore saver!